Tests/FeaturevisorSDKTests/InstanceTests.swift

```diff
@@ -4,6 +4,21 @@ import XCTest
 @testable import FeaturevisorTypes
 
 class FeaturevisorInstanceTests: XCTestCase {
+    
+    func testEncodeDoesNotThrow() throws {
+        let evaluation = Evaluation(
+                    featureKey: "feature123",
+                    reason: .allocated,
+                    bucketValue: 42,
+                    ruleKey: "rule456",
+                    enabled: true,
+                    variableKey: "color"
+                )
+                
+        let encoder = JSONEncoder()
+        
+        XCTAssertNoThrow(try encoder.encode(evaluation))
```

Comment:
Not throwing an error doesn't mean that the encoding was done correctly. Can we make more detailed test?

Reply:
I wanted to but evaluation would have to also conform to decodable which I didn't want to do only to test it. Unless there are some other ways